File: Gollala/app/src/main/java/com/wisefashion/mdlens/util/ProgressUtil.java
package com.wisefashion.mdlens.util;

import android.app.Dialog;
import android.content.Context;
import android.view.Window;

import com.wisefashion.mdlens.R;

/**
 * Created by skcrackers on 5/27/16.
 */
public class ProgressUtil extends Dialog {
    public ProgressUtil(Context context) {
        super(context);
        requestWindowFeature(Window.FEATURE_NO_TITLE);
        setContentView(R.layout.view_progress_util);
    }
}

File: 2025/Sayrobin/app/src/main/java/com/creator/sayrobin/util/ProgressUtil.kt
package com.creator.sayrobin.util

import android.app.Dialog
import android.content.Context
import android.view.Window
import com.creator.sayrobin.R

/**
 * Created by skcrackers on 5/27/16.
 */
class ProgressUtil(context: Context?) : Dialog(context!!) {
    init {
        requestWindowFeature(Window.FEATURE_NO_TITLE)
        setContentView(R.layout.view_progress_util)
    }
}